src/messages/en.json

```diff
@@ -2,5 +2,60 @@
     "Index": {
       "title": "Home",
       "description": "Testing internationalization."
+    }, 
+
+    "CommitteeCards": {
```

Comment:
put this within a json object for the committees page. so the whole file looks like:
``` 
{ "Index": {...},
  "Committees" : {
        "CommitteeCards": {...}
   }
} 
```
because there are other components within the page and we're trying to get a nesting structure going for easy navigation inside the file when the translations for all the pages are on here.